event_sourcery_sqlalchemy/sqlalchemy_outbox.py

```diff
@@ -56,6 +58,7 @@ def outbox_entries(
                 ),
             )
             for entry in entries
+            if self._filterer(entry.data)
```

Comment:
Filtering in SQL strategy filtering should happen before inserting. Otherwise, we'll be left with unsent events that will be stuck in the outbox_entries table

Reply:
We could leave it in both places to make sure filtering is applied once something applies some criteria on a working system, but that wouldn't bring too many benefits.

Normally, entries in the outbox table will be kept for a very short amount of time, so gaining any extra filtering gives little advantage.